sound/soc/sof/trace.c

```diff
@@ -544,10 +539,7 @@ void snd_sof_release_trace(struct snd_sof_dev *sdev)
 	if (!sdev->dtrace_is_supported || !sdev->dtrace_is_enabled)
 		return;
 
-	ret = snd_sof_dma_trace_trigger(sdev, SNDRV_PCM_TRIGGER_STOP);
-	if (ret < 0)
-		dev_err(sdev->dev,
-			"error: snd_sof_dma_trace_trigger: stop: %d\n", ret);
+	snd_sof_dma_trace_trigger(sdev, SNDRV_PCM_TRIGGER_STOP);
```

Comment:
check return value?

Reply:
The error handling has already been done in the snd_sof_dma_trace_trigger() in this case. So we don't need the return value any more here.

Reply:
I see you moved the error print there.